pkg/reconciler/build/resources/pod.go

```diff
@@ -277,12 +281,26 @@ func MakePod(build *v1alpha1.Build, kubeclient kubernetes.Interface) (*corev1.Po
 				return nil, err
 			}
 			// Prepend the custom container to the steps, to be augmented later with env, volume mounts, etc.
+
+			// initContainers = append(initContainers, *cust)
+			// TODO(aaron-prindle) not if I correctly handled  this piece....
```

Comment:
looking at the `Git` and `GCS` cases, I don't think this block needs to change? I think this is just for a custom method of retrieving source data to operate on